services/ui-src/src/components/overlays/EntityDetailsDashboardOverlay.test.tsx

```diff
@@ -1,14 +1,20 @@
-import { render, screen } from "@testing-library/react";
+import { fireEvent, render, screen } from "@testing-library/react";
```

Comment:
suggestion: use `userEvent` instead of `fireEvent`. According to React Testing Library it better simulates a real user interaction (e.g. button click rather than dom event)

We have `fireEvent` a few places throughout our app so this is more just a suggestion for future changes. Not a big deal to update here. Just sharing!

[react testing library docs for userEvent](https://testing-library.com/docs/user-event/intro/#differences-from-fireevent)